queries/models/subsidio/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - subsidio
 
+## [2.1.0] - 2025-10-22
+
+### Alterado
+
+- Altera nome da CTE `temperatura_inmet` para `temperatura_inmet_alertario` no modelo `aux_viagem_temperatura` e troca refência para utilizar o modelo `temperatura` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/977)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)